hq/app/AppComponents.scala

```diff
@@ -112,6 +114,10 @@ class AppComponents(context: Context)
     .build()
   private val securityCenterSettings = SecurityCenterSettings.newBuilder().setCredentialsProvider(Config.gcpCredentialsProvider(configuration)).build()
   private val securityCenterClient = SecurityCenterClient.create(securityCenterSettings)
+  private val dynamoDbClient: AmazonDynamoDB = AmazonDynamoDBClientBuilder.standard()
+    .withCredentials(securityCredentialsProvider)
+    .withRegion(Config.region.getName)
+    .build()
```

Comment:
I think at this point you have the 'stage' property available (either in `Config` or `configuration`) which you could use to work out whether to build a local dynamo client or this one.